src/main/config/azure-application-local-runner.properties

```diff
@@ -21,29 +21,15 @@ job.coordinator.factory=org.apache.samza.zk.ZkJobCoordinatorFactory
 job.default.system=eventhubs
 job.coordinator.zk.connect=localhost:2181
 
-# Azure EventHubs System
-systems.eventhubs.samza.factory=org.apache.samza.system.eventhub.EventHubSystemFactory
-systems.eventhubs.stream.list=output-stream,input-stream
+# Add your EventHubs input stream credentials here,
+# or comment them out here and define them in AzureApplication
+sensitive.streams.input-stream.eventhubs.sas.keyname=YOUR-SAS-KEY-NAME
```

Comment:
I think these should not be in the streams.stream-id scope anymore since these will be read and generated by the descriptor.

Reply:
Do you mean we should omit this property in the example since it is sufficient that it is included in the app already?

Reply:
The right way to do these would be to only set these in configurations and remove them from the descriptor. Setting this using descriptor will generate the same config but without the sensitive prefix.